### ISSUE TITLE
Advantages of "softmax_helper" over original "torch.nn.functional.softmax"?

### ISSUE THREAD

**Issue by EricKani** [MIC-DKFZ/nnUNet#273]

**Comment by FabianIsensee** [comment]
None, way back when I implemented it the pytorch implementation would not work with n-d tensors. Now it does and I am considering changing it to the torch implementation in the future.
Best,
Fabian
